Classes/TagHelper.php

```diff
@@ -18,8 +18,12 @@ class TagHelper
     const DATA_NEVER_BLOCK = "data-never-block";
 
     // Type Constants
-    const TEXT_PLAIN = "text/plain";
-    const TEXT_JAVASCRIPT = "text/javascript";
+    const TYPE_TEXT_PLAIN = "text/plain";
+    const TYPE_APPLICATION_JSON_LD = "application/ld+json";
+
+    // TODO: switch to "text/javascript" at some point
```

Comment:
I do not understand this comment :)

Reply:
Will fix later and refactor to use "application/javascript" instead